### PR TITLE
Added Toast

### LINKED ISSUE #1: Add a Toast
Add a Toast Message when the time is successfully set.

### PR DESCRIPTION
**Pull request by ekthapriyesh24**
fixed #1 

- Shows **Time Set**! toast on pressing save button

### CODE REVIEW

File: app/src/main/java/com/example/android/timepicker/MainActivity.java [hunk 1]
```diff
@@ -13,13 +14,13 @@
...
     private String format="";
-
+    private Toast toast;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        timePicker1=(TimePicker)findViewById(R.id.timePicker1);
-        time=(TextView)findViewById(R.id.textView1);
+        timePicker1=findViewById(R.id.timePicker1);
```
Comment (harchani-ritik) [inline app/src/main/java/com/example/android/timepicker/MainActivity.java]: @ekthapriyesh24 Why you have removed the typecasting?

**Comment by harchani-ritik** [comment]
@ekthapriyesh24 squash all commits into one and then open a new pull request.

File: app/src/main/java/com/example/android/timepicker/MainActivity.java (hunk 1 above)
Comment (ekthapriyesh24) [reply app/src/main/java/com/example/android/timepicker/MainActivity.java]: **Type casting are no longer required,got a warning to remove them while commiting from studio**

Screenshot from 2019-03-21 20-07-13

**Comment by harchani-ritik** [comment]
Okay Then!

**Pull request merged**